frontend/hub/collections/components/CodeBlock.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/CodeBlock/code-block';
+import { css } from '@patternfly/react-styles';
+
+export interface CodeBlockProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the code block */
+  children?: React.ReactNode;
+  /** Additional classes passed to the code block wrapper */
+  className?: string;
+  /** Actions in the code block header. Should be wrapped with CodeBlockAction. */
+  actions?: React.ReactNode;
+}
+
+export const CodeBlock: React.FunctionComponent<CodeBlockProps> = ({
```

Comment:
Are we using this component? Could we just leverage the native PF ones?

Reply:
You are right. It seems my IDE even imported react version of CodeBlock, so Im not using this CodeBlock from old code. I deleted this component. I dont see any reason why not use classic react CodeBlock.